brightwind/analyse/plot.py

```diff
@@ -169,22 +169,185 @@ def plot_monthly_means(data, coverage=None, ylbl=''):
     return ax.get_figure()
 
 
-def plot_timeseries(data, date_from='', date_to='', y_limits=(None, None)):
+def _timeseries_subplot(x, y, x_label=None, y_label=None, x_limits=None, y_limits=None, x_tick_label_angle=25,
+                        line_marker_types=None, line_colors=COLOR_PALETTE.color_list, subplot_title=None,
+                        legend=True, ax=None):
+    """
+    Plots a scatter subplot between the inputs x and y. The trendline_y data and the line of slope 1 passing through
+    the origin are also shown if provided as input of the function.
```

Comment:
@BiancaMorandi this docstring is for a scatter plot with a regression line. Are you able to update it?

Reply:
this is done